shared/src/test/diff/ucs/LitUCS.mls

```diff
@@ -4,35 +4,37 @@
 module A
 //│ module A()
 
-// FIXME
+// This one is easy to fix but what about the next one?
+// The following example can better reveal the essence of the problem.
 fun test(x: 0 | A) = if x is
   0 then 0
   A then A
-//│ ╔══[ERROR] Type mismatch in application:
-//│ ║  l.8: 	fun test(x: 0 | A) = if x is
-//│ ║       	                        ^
+//│ fun test: (x: 0 | A,) -> (0 | A)
+
+// FIXME
```

Comment:
Why is this a FIXME?

Reply:
The example below still doesn't work.

Reply:
This error is expected, as I've explained you on Telegram... `===` isn't meant to compare unrelated types and you're comparing `0 | A` to `0`, which implies comparing `A` with `0`.

Reply:
I see! Thanks!